layouts/header.html

```diff
@@ -9,6 +9,12 @@
     </div><!-- /.header-wrapper -->
   </div><!-- /#top-header -->
 
+  <div id="notification-header">
+    <div class="header-wrapper">
+      <marquee style="color: #FF0000;"> We migrated to Oracle Cerner sites. New Oracle Cerner site will be availbale shortly. </marquee>
```

Comment:
Please make the statement as "Soon we will be migrating this site content to Oracle Help Center."

Reply:
added ab96d526f7b84342c47b058410366c2fa519f03b